src/components/ErrorBoundary.js

```diff
@@ -15,6 +18,27 @@ class ErrorBoundary extends Component {
 
 	static contextType = StoreContext
 
+	refreshFights = () => {
+		const {reportStore} = this.context
+		reportStore.refreshReport()
+	}
+
+	renderRefreshButton = (error) => {
+		if (error instanceof NotFoundError) {
+			return <Header>
+				<div className={styles.refreshheader + ' pull-right'}>
+					<span className={styles.refresh} onClick={this.refreshFights}>
```

Comment:
Doesn't need doing _now_, but this would be a prime candidate to pull out into a shared component (inc. the click handling)